### DISCUSSION TITLE
Opposition (Vs. D&B Definitive Edition) Modchart (Psych Engine)

### DISCUSSION THREAD

**Discussion by randomaccount20** [DEMOLITIONDON96/Demolition-Engine#84] [Lua Q&A]
So im just wondering, how can i get the strumlines modchart from opposition (Vs. D&B Definitive Edition)? I can't solve it by guessing. but LUA for psych engine? It's just that i got banned in https://github.com/mayo78/PSYCHDISCUSSIONS/discussions and can't fucking question the same goddamn answer, i just need the strumlines going up and down like in the same gameplay of that song.

**Comment by RealDealJinx** [comment]
wdym

**Reply by randomaccount20** [reply]
Its that the strikeline moves up and down, while the arrows move in a whirly mess.

**Reply by randomaccount20** [reply]
Oh and also, i need it on psych engine

**Comment by ThatOneSillyGuyDon** [comment]
honestly, idk how to do this, maybe I can at least try, then again, i fucking suck at lua :P
